src/test/java/org/junitpioneer/jupiter/issue/IssueExtensionIntegrationTests.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter.issue;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestReporter;
+import org.junit.platform.engine.discovery.DiscoverySelectors;
+import org.junit.platform.launcher.core.LauncherDiscoveryRequestBuilder;
+import org.junit.platform.launcher.core.LauncherFactory;
+import org.junitpioneer.jupiter.Issue;
+import org.junitpioneer.jupiter.IssueTestSuite;
+import org.opentest4j.TestAbortedException;
+
+/**
+ * Mary Elizabeth Fyre: Do Not Stand at My Grave and Weep is in the public domain.
+ */
+public class IssueExtensionIntegrationTests {
+
+	private final StoringIssueProcessor issueProcessor = (StoringIssueProcessor) IssueProcessorFactory
```

Comment:
So you try to convert every IssueProcessoir which is found into a StoringIssueProcessor?

Reply:
It's our own tests, so I made the bold assumption that we will only use the one `IssueProcessor`. 😛

Reply:
I'm fairly certain this won't change in the future, but if it does we can update the test?